recipes/bodo/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "bodo" %}
+{% set version = "2024.12.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bodo-ai/Bodo/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 01167655812303bf8d878a111d54a5d19fa8e7f438d968f60a9a9ff8378076cb
+
+build:
+  skip: True  # [win]
+  number: 0
+  rpaths:
+    - lib
+    - {{ PREFIX }}/lib
+    - {{ SP_DIR }}/pyarrow
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+
+  host:
+    - python
+    - pip
+
+    # Build Dependencies
+    - setuptools >=64
+    - cython ~=3.0
+    - setuptools_scm >=8
+    - scikit-build-core
+    - cmake
+    - ninja
+    - mpi4py =3.1.5
+
+    # Arrow Deps
+    - libarrow =18.1.0
+    - libarrow-dataset =18.1.0
+    - libparquet =18.1.0
+    - pyarrow-core =18.1.0
+
+    - numba =0.60.0
+    # Compatible with Numba requirements.
+    # Resolves build issues on nightly
+    - numpy >=1.24,<1.27
+    - pandas >=2.2,<2.3
+    - fsspec >=2021.09
+    - libboost-devel =1.85
```

Comment:
Use the libboost-headers package since it doesn't look like your are linking to libboost.

There are also overdepending warnings from conda-build about zstd and curl.